base_geoengine/geo_field.py

```diff
@@ -143,7 +130,8 @@ def create_geo_column(self, cursor, col_name, geo_column, table, model):
         return True
 
     def update_geo_column(self, cursor, col_name, geo_column, table, model):
-        """Update a column of type the geom does. !! not do a lot of test yet"""
+        """Update a column of type the geom does. !! not do a lot of test yet
+        """
         query = ("SELECT srid, type, coord_dimension "
```

Comment:
All the queries I've seen so far use triple quotes, this one is different, but it doesn't need to.

The advantage of triplestrings is that you avoid the issue of forgetting to append a space.